boards/arm/adafruit_trinket_m0/doc/adafruit_trinket_m0.rst

```diff
@@ -0,0 +1,147 @@
+with an onboard RGB LED, USB port, and range of I/O broken out onto 5
+pins.
+
+.. image:: img/adafruit_trinket_m0.png
+     :width: 500px
+     :align: center
+     :alt: Adafruit Trinket M0
+
+Hardware
+********
+
+- ATSAMD21E18A ARM Cortex-M0+ processor at 48 MHz
+- 256 KiB flash memory and 32 KiB of RAM
+- Internal trimmed 8 MHz oscillator
+- A user LED
+- A RGB DotStar LED
+- Native USB port
+- One reset button
+
+Supported Features
+==================
+
+The adafruit_trinket_m0 board configuration supports the following hardware
+features:
+
++-----------+------------+--------------------------------------+
+| Interface | Controller | Driver/Component                     |
++===========+============+======================================+
+| NVIC      | on-chip    | nested vector interrupt controller   |
++-----------+------------+--------------------------------------+
+| Flash     | on-chip    | Can be used with NFFS to store files |
++-----------+------------+--------------------------------------+
+| SYSTICK   | on-chip    | systick                              |
++-----------+------------+--------------------------------------+
+| WDT       | on-chip    | Watchdog                             |
++-----------+------------+--------------------------------------+
+| GPIO      | on-chip    | I/O ports                            |
++-----------+------------+--------------------------------------+
+| USART     | on-chip    | Serial ports                         |
++-----------+------------+--------------------------------------+
+| SPI       | on-chip    | Serial Peripheral Interface ports    |
++-----------+------------+--------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig
+:file:`boards/arm/adafruit_trinket_m0/adafruit_trinket_m0_defconfig`.
+
+Connections and IOs
+===================
+
+The `Adafruit Learn`_ site has detailed information about the board
+including `pinouts`_ and the `schematic`_.
+
+System Clock
+============
+
+The SAMD21 MCU is configured to use the 8 MHz internal oscillator
+with the on-chip PLL generating the 48 MHz system clock.  The internal
+APB and GCLK unit are set up in the same way as the upstream Arduino
+libraries.
+
+Serial Port
+===========
+
+The SAMD21 MCU has 6 SERCOM based USARTs.  On the Trinket, SERCOM0 is
+the Zephyr console and is available on pins 3 (RX) and 4 (TX).
+SERCOM2 is available on pins 2 (RX) and 0 (TX).
+
+SPI Port
+========
+
+The SAMD21 MCU has 6 SERCOM based SPIs.  On the Trinket, SPI1 is used
+to drive the DotStar RGB LED.  SERCOM0 can be put into SPI mode and
+used to connect to devices over pin 2 (MISO), pin 4 (MOSI), and pin 3
+(SCK).
+
+Programming and Debugging
+*************************
+
+The Trinket M0 ships the BOSSA compatible UF2 bootloader.  The
+bootloader can be entered by quickly pressing the reset button twice.
```

Comment:
You use the term ``tapping`` below, I like that better since it implies a press and release:

    by quickly tapping the reset button twice.

Reply:
Done